gen/dcos-config.yaml

```diff
@@ -379,6 +379,23 @@ package:
         ],
         "enabled": "{{ telemetry_enabled }}"
       }
+  - path: /etc/dcos-metrics-config.yaml
+    content: |
+      collector:
```

Comment:
That piece of configuration is set via the command line flag `--role <agent,master>`. 

I'm not sure what you mean by "just defaults". The collector is broken down into discrete collectors and producers. Today, we have a node, mesos agent, and application (container metrics sent from metrics module) collector. We have a single producer for HTTP, but that will increase in 1.10. So this config file reflects that design.